erpnext/stock/doctype/item/item.json

```
@@ -1054,15 +1055,21 @@
    "fieldtype": "Data",
    "label": "Default Manufacturer Part No",
    "read_only": 1
+  },
+  {
+   "fieldname": "website_image_alt",
+   "fieldtype": "Data",
+   "label": "Image Alt Text"
```

```suggestion
   "label": "Image Alternative Text"
```
A layperson should also understand it

---

`Alt` is a generally accepted term. "Image Alternative Text" is not a clear label as well. Lets' make it Image Description